### PR TITLE
Rename to Minetest Game (fixes #480)

### LINKED ISSUE #480: Actually rename Minetest (the subgame)
This issue is a follow-up of #437.

Long story short, it has been decided to rename this subgame to “Minetest Game”.

Now since we got that done, the subgame has not been actually renamed yet.

A couple of files in the subgame must be changed. The relevant files should use the new name “Minetest Game” constantly. Mentions of “minetest_game” or even “Minetest” (in the sense of “subgame”) should be removed or replaced by “Minetest Game”. I suggest that from now on, “minetest_game” should only be used to refer to this repository. It should be (lightly) discouraged to use this term to refer to this subgame. The term “Minetest” alone should *not* be used anymore to refer to this subgame.

The following files need to be updated:

* `menu/header.png` (new or altered image must be drawn)
* `game.conf`
* `README.txt`
* `game_api.txt`
* `minetest.conf.example`
* (mention any files I forgot)

Additionally, the desciption of this repository on GitHub should be changed as well.

TODO list for your convenience:
* [x] Change `menu/header.png`
* [x] Change `game.conf`
* [x] Change `README.txt`
* [x] Change`game_api.txt`
* [x] Change `minetest.conf.example`
* [x] Check out other readme files of mods and change them if needed
* [x] Make sure the name “Minetest Game” is used consistently throughout this subgame
* [x] Change repo description on GitHub

Edit: After all or most of this has been done, also post a thread into the News forum:
https://forum.minetest.net/viewforum.php?f=18

* [x] Post a thread into the news forum

Edit 2: And the wikis, too.
* [x] Update the developer wiki
* [x] Update the community wiki

### PR DESCRIPTION
**Pull request by BlockMen**
Write here if there is anything missing.

EDIT: or wrong

### CODE REVIEW

File: mods/default/README.txt [hunk 1]
```diff
@@ -71,7 +68,8 @@ Calinou (CC BY-SA):
   default_glass_detail.png
 
 MirceaKitsune (WTFPL):
-  character.x
+  character.obj
```
Comment (kilbith) [inline mods/default/README.txt:71]: `character.b3d`

File: mods/vessels/README.txt [hunk 2]
```diff
@@ -1,31 +1,14 @@
-Minetest 0.4 mod: vessels
+Minetest Game mod: vessels
 ==========================
 
-Crafts
--------
-Glass bottle (yields 10)
+Adds glass bottle, drinking glass, steel bottle and vessles shelf.
```
Comment (kilbith) [inline mods/vessels/README.txt:4]: typo

**Comment by Wuzzy2** [comment]
Change “Minetest Engine” to “Minetest” or “the Minetest engine”. Otherwise, “Minetest Engine” looks like a name, but this “name” is not official and to my knowledge it is only used in this subgame's readme files, so it's inconsistent.

Apart from that, I am okay with this PR. :-)

**Comment by ShadowNinja** [comment]
A photorealistic title belongs in something like HDX.  The title should be much simpler, and probably very low res.

**Comment by kilbith** [comment]
@BlockMen Are you still alive ?

**Pull request closed without merging**